### PR TITLE
READY FOR REVIEW - LoA updates

### PR DESCRIPTION
**Pull request by swithinfoote**
A number of updates to address [issue 97](https://github.com/tewhatuora/api-standards/issues/97) as well as a number of other inconsistencies in the LoA section.

### CODE REVIEW

File: docs/api-security/10-LevelofAssurance.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ title: Level of Assurance
 Content is draft and in review – this content may change until review is complete and formally published.
 :::
 
-Level of Assurance (LoA) has been defined to support the OpenID Connect API security framework. OpenID Connect introduces the concept of identity claims about the end user. The Level of Assurance claim is required to:
+Level of Assurance (LoA) has been defined to support the OpenID Connect API security framework. OpenID Connect introduces the concept of identity claims about the end user.
+
+LoA in OIDC refers to the certainty an API provider can have about the identity of the health sector participant accessing their service. It's determined by how strong the authentication was and the verification methods used by the identity provider. Higher LoA means stronger verification. OIDC uses the Authentication Context Class Reference `acr` token claim to communicate LOA levels to API providers.
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:11]:
```suggestion
LoA in OIDC refers to the certainty an API Provider can have about the identity of the Health Sector Participant accessing their service. It's determined by how strong the authentication was and the verification methods used by the identity provider. Higher LoA means stronger verification. OIDC uses the Authentication Context Class Reference `acr` token claim to communicate LOA levels to API Providers.
```

File: docs/api-security/10-LevelofAssurance.md [hunk 2]
```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
 
 - Ensure the API consumer has the appropriate authorisation to act on behalf of the end user
 
-API providers **SHOULD** adopt a Level of Assurance model which can be applied when API consumers are accessing health records that are classed as MEDICAL IN-CONFIDENCE.
+API providers **MUST** adopt a Level of Assurance model which can be applied when API consumers are accessing health records that are classed as MEDICAL IN-CONFIDENCE.
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:27]:
```suggestion
API Providers **MUST** adopt a Level of Assurance model which can be applied when API Consumers are accessing health records that are classed as MEDICAL IN-CONFIDENCE.
```

File: docs/api-security/10-LevelofAssurance.md [hunk 3]
```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
...
+
+API providers **MUST** indicate the LoA associated with a health sector participants authentication and authorisation process in any issued tokens.
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:29]:
```suggestion
API Providers **MUST** indicate the LoA associated with a Health Sector Participant's authentication and authorisation process in any issued tokens.
```

File: docs/api-security/10-LevelofAssurance.md [hunk 4]
```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
 
 - Ensure the API consumer has the appropriate authorisation to act on behalf of the end user
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:25]:
```suggestion
- Ensure the API Consumer has the appropriate authorisation to act on behalf of the end user
```

File: docs/api-security/10-LevelofAssurance.md [hunk 5]
```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
...
+API providers **MUST** indicate the LoA associated with a health sector participants authentication and authorisation process in any issued tokens.
+
+### Level of Assurance
+
+Issued tokens can include the following claims that relate to the Level of Assurance (LoA) of the user authentication.
 
-API providers **MUST** indicate the LoA associated with a health sector participants authentication and authorisation process in any issued access tokens.
+The LoA captures the strength of the authentication method used and provides an indication of the trust that the user's identity is correct.
+
+The API Consumer uses these claims to confirm the level of trust that the user is who they say they are and based on this what level of grant.
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:37]:
```suggestion
The API Provider uses these claims to confirm the level of trust that the user is who they say they are, and based on this, what level of access should be granted.
```

File: docs/api-security/10-LevelofAssurance.md [hunk 6]
```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
...
+
+|Claim| Description| How Applied|
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:39]:
```suggestion
|Claim| Description| How They Are Applied|
```

File: docs/api-security/10-LevelofAssurance.md [hunk 7]
```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
...
+|---|---|---|
+|`acr`|Authentication Context Class Reference| LoA based on a pre-defined set of values, the higher the value the stronger the LoA<br/>&nbsp;&nbsp;&nbsp;&nbsp;(i) `0` = no authentication performed **MAY** be applied to UNCLASSIFIED information. **MUST NOT** be applied to MEDICAL IN-CONFIDENCE information<br/>&nbsp;&nbsp;&nbsp;&nbsp;(ii) `1` = credential only. **MAY** be applied to UNCLASSIFIED information. **MUST NOT** be applied to MEDICAL IN-CONFIDENCE information<br/>&nbsp;&nbsp;&nbsp;&nbsp;(iii) `2` = credential and password and OTP. **MAY** be applied to UNCLASSIFIED information. **MUST NOT** be applied to MEDICAL IN-CONFIDENCE information<br/>&nbsp;&nbsp;&nbsp;&nbsp;(iv) `3` = Password, OTP and Multi factor Authentication required. **MUST** be applied to MEDICAL IN-CONFIDENCE information|
+|`amr`|Authentication Methods References|Details the authentication method used e.g.<br/>-    password<br/>-  OTP<br/>-   Biometrics<br/><br/> API providers can define the requirements (acr) based on this information which are enforced by the API consumers  |
```
Comment (kyle-mwnz) [inline docs/api-security/10-LevelofAssurance.md:42]:
```suggestion
|`amr`|Authentication Methods References|Details the authentication method used e.g.<br/>-    password<br/>-  OTP<br/>-   Biometrics<br/><br/> API Providers can define the requirements (acr) based on this information which are enforced by the API Consumers  |
```

File: docs/api-security/10-LevelofAssurance.md (hunk 5 above)
Comment (kyle-mwnz) [reply docs/api-security/10-LevelofAssurance.md:37]: Unless I misunderstand the intent, this should be the API Provider?

**Review by kyle-mwnz — COMMENTED** [review]
Mostly formatting suggestions, but one content change to confirm before approval

File: docs/api-security/10-LevelofAssurance.md (hunk 5 above)
Comment (ChrisSquats) [reply docs/api-security/10-LevelofAssurance.md:37]: I concur

**Review by ChrisSquats — APPROVED** [review]

**Review by kyle-mwnz — APPROVED** [review]

**Pull request merged**